addons/mail/models/mail_cc_mixin.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import _, api, fields, models, tools
+
+
+class MailCCMixin(models.AbstractModel):
+    _name = 'mail.cc.mixin'
+    _inherit = 'mail.thread'
+    _description = 'Email CC management'
+
+    email_cc = fields.Char('Email cc', help='List of cc from incoming emails.')
+
+    def _email_dict(self, cc_string):
```

Comment:
Maybe update naming to be more explicit ? Like _mail_cc_sanitized_dict or stuff like that. Mail_cc being the standard prefixing methods by mixin name thingy.